### PR TITLE
fix(executor): GODEBUG=x509ignoreCN=0

### PR DESCRIPTION
**Pull request by alexec**
To test this,lets merge it and make sure https://workflows.apps.argoproj.io/ can run coin flip

### CODE REVIEW

File: workflow/controller/workflowpod.go [hunk 1]
```diff
@@ -535,6 +535,13 @@ func (woc *wfOperationCtx) createEnvVars() []apiv1.EnvVar {
 			Name:  common.EnvVarContainerRuntimeExecutor,
 			Value: woc.getContainerRuntimeExecutor(),
 		},
+		// This flag was introduced in Go 16 and will be removed in Go 16.
```
Comment (whynowy) [inline workflow/controller/workflowpod.go:538]: xxx 16 and xxxx 16?

**Review by whynowy — APPROVED** [review]

File: workflow/controller/workflowpod.go (hunk 1 above)
Comment (alexec) [reply workflow/controller/workflowpod.go:538]: I will fix this

**Comment by alexec** [comment]
Next steps. 

* [ ] Wait for new image.
* [ ] Run coinflip

**Comment by alexec** [comment]
@simster7 @sarabala1979 this appears to have fixed the issue on GCP.

**Pull request merged**